documentation/how-to/pg-service.en.md

```diff
@@ -23,3 +23,73 @@ Usually the path on Android devices looks something like this: `/Android/data/ch
 QFieldCloud support `pg_service.conf` configurations too. You need to configure your PostgreSQL layers with "Offline editing" cloud action and store your service settings on QFieldCloud Project's Secrets page.
 
 Read more [how to configure PostgreSQL service](../reference/qfieldcloud/secrets.md) in the QFieldCloud documentation.
+
+## Creating a `pg_service.conf` File for PostgreSQL Connection in QGIS and Secrets
+
+Before beginning, ensure that you have allowed connections to QFieldCloud. Refer to [Technical specs](../reference/qfieldcloud/specs.md) for instructions.
+
+1. **Create a Text File**: Open your preferred text editor and create a new text file to define PostgreSQL service connection parameters. Save this file in your user's home directory, typically `~/.pg_service.conf`.
```

Comment:
"home directory" and "~" are unix centric. Can we add instructions for windows users?

Reply:
Yes, is already added the steps for the Windows users